api/python/tests/integration/test_packages.py

```diff
@@ -462,7 +474,7 @@ def add_pkg_file(pkg, lk, filename, data, *, version):
         )
         with patch('time.time', return_value=timestamp1), \
              patch('quilt3.data_transfer.MAX_CONCURRENCY', 1):
-            remote_pkg = new_pkg.push(pkg_name, registry)
+            remote_pkg = new_pkg.push(pkg_name, registry, force=True)
```

Comment:
Why do we need `force=True` here?

Reply:
Because otherwise, it makes extra S3 calls, so I'd need to mock those. I guess `setup_s3_stubber_push_manifest` could be updated to mock them by default, but `force=True` plus a new unit test for push conflicts seemed simpler.

Reply:
> but force=True plus a new unit test for push conflicts seemed simpler

Well, yes, it's simpler, but are we going to use `force=True` in every new test?